components/TestimonyCard/TestimonyItem.tsx

```diff
@@ -76,6 +74,24 @@ export const TestimonyItem = ({
 
   const { t } = useTranslation("testimony")
 
+  const IconSpacer = () => {
```

Comment:
Not a blocker, but if we're going to take this approach for now, should we at least remove the alt text (so that if the `edit-testimony.svg` doesn't load, we won't see "Edit Icon" text on each of these buttons?) Ideally we should accomplish this with flex spacing / min heights - maybe something to revisit in the accessibility work if our audit complains about the alt text?

Reply:
you're right, using "p-" at different values to balance out the elements is a much cleaner design